oak/common/app_config.bzl

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2020 The Project Oak Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Rule for serializing an Oak application configuration."""
+
+def serialized_config(name, textproto, modules):
+    """Serializes an Oak application configuration in a binary file.
+
```

Comment:
Can you mention what the implicit output name is? Otherwise I think it's impossible for clients to figure it out, apart from looking at this implementation.

See https://docs.bazel.build/versions/2.0.0/be/c-cpp.html#cc_binary_implicit_outputs

Reply:
Done, using the `name` argument as the name of the output file (without `.bin`)